pkg/stanza/operator/helper/transformer.go

```diff
@@ -5,6 +5,7 @@ package helper // import "github.com/open-telemetry/opentelemetry-collector-cont
 
 import (
 	"context"
+	goerrors "errors"
```

Comment:
Why don't we use `"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/errors"`

Reply:
I use the standard package's [errors.Join](https://pkg.go.dev/errors#Join) function and Stanza errors package does not have this as far as I'm aware.

Reply:
How about add this function to `stanza/errors`? I think stanza provide its own errors package for some reason, for example may be all errors have some common format, but I'm not sure about this.

@djaglowski WDYT?